target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-rt-ac42u.dts

```diff
@@ -0,0 +1,302 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-ap.dk04.1.dtsi"
```

Comment:
This was from ASUS's device dts right?

This would be Ok, if the ASUS ACRH17/AC42U was really based on the Qualcomm reference board. Please include ``` #include "qcom-ipq4019.dtsi" ``` instead. 

Note: (This should also get rid of SPI and the secondary uart which you probably are not using)

Reply:
Done